From: =?UTF-8?q?Lum=C3=ADr=20=27Frenzy=27=20Balhar?=
 <frenzy.madness@gmail.com>
Date: Tue, 28 Jan 2025 14:53:51 +0100
Subject: [PATCH] chore: remove upper version limit from packaging (#40)

Packaging uses calendar versioning so there is no need to limit the upper version.

diff --git a/setup.py b/setup.py
index 7b662c3..ad16860 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ setup(
     include_package_data=True,
     py_modules=["vvm"],
     python_requires=">=3.8, <4",
-    install_requires=["requests>=2.32.3,<3", "packaging>=23.1,<25"],
+    install_requires=["requests>=2.32.3,<3", "packaging>=23.1"],
     license="MIT",
     zip_safe=False,
     keywords="ethereum vyper",
